src/main/java/com/suse/saltstack/netapi/config/ClientConfig.java

```diff
@@ -12,6 +12,25 @@
     public static final Key<URI> URL = new Key<>(URI.create("http://localhost:8000"));
     public static final Key<String> TOKEN = new Key<>();
 
+    // Request timeout settings
```

Comment:
I don't think we need this single line comment since there is the javadoc one below.